ci/jjb/jobs/pulp-restore.yaml

```diff
@@ -0,0 +1,43 @@
+- job-template:
+    name: 'pulp-{pulp_version}-{pulp_build}-restore-{os}'
+    node: '{os}-np'
+    properties:
+        - qe-ownership
+    scm:
+        - pulp-packaging-github
+    wrappers:
+        - config-file-provider:
+            files:
+                - file-id: rhn_credentials
+                  variable: RHN_CREDENTIALS
+        - inject:
+            properties-content: |
+                OS={os}
+                PULP_BUILD={pulp_build}
+                PULP_VERSION={pulp_version}
+    triggers:
+        - reverse:
+            jobs: 'pulp-2.11-stable-backup-rhel6'
```

Comment:
If I add a new variable, will I have to put it in the job name? I ran into an issue where the pulp_build would not get unpacked out of list unless it was in the job name. So it was resolving as `['stable']` instead of `'stable'`